drivers/mega/meta.go

```diff
@@ -9,8 +9,10 @@ type Addition struct {
 	// Usually one of two
 	//driver.RootPath
 	//driver.RootID
-	Email    string `json:"email" required:"true"`
-	Password string `json:"password" required:"true"`
+	Email       string `json:"email" required:"true"`
+	Password    string `json:"password" required:"true"`
+	TwoFACode   string `json:"two_fa_code" required:"false"`
```

Comment:
You can add the help message as a help tag here, and then the language files will be automatically generated.

Reply:
help message has been added and the pr in alist-web is closed